work_queue/src/work_queue_json.c

```diff
@@ -0,0 +1,284 @@
+#ifndef WORK_QUEUE_JSON_H
+#define WORK_QUEUE_JSON_H
+
+#include "work_queue_json.h"
+#include "jx_parse.h"
+#include "jx_print.h"
+#include "work_queue.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+char* wq[] = {"name", "port", "priority", "num_tasks_left", "next_taskid", "workingdir", "master_link", 
```

Comment:
Since this is a constant and private to the module, make it `static const char *`
And be generous with the names of things for the sake of later readers.
Call this `work_queue_allowed_properties` or something like that.